dll/shellext/zipfldr/lang/ro-RO.rc

```diff
@@ -13,10 +13,10 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 CAPTION "Selectați o destinație"
 FONT 8, "MS Shell Dlg", 400, 0, 0x0
 BEGIN
-    LTEXT           "Selectați folderul de destinație",IDC_STATIC,86,12,154,8
+    LTEXT           "Selectaţi un folder unde vor fi extrase fişierele.",IDC_STATIC,86,12,154,8
```

Comment:
does that fit the button? or is it cut off by the 154?

Reply:
You're right. I made a mistake. I will revert this change.

Reply:
Resolved, thanks.